aip/general/0121.md

```diff
@@ -61,6 +61,10 @@ exposes a large number of resources with a small number of methods on each
 resource. The methods can be either the standard methods ([Get][], [List][],
 [Create][], [Update][], [Delete][]), or [custom methods][].
 
+If a request and response in a standard method contains the resource (such as
+the request of [Create][] or the response of [Get][]), the schema (proto
+message) for the resource **must** be identical.
+
```

Comment:
If the request to or the response from a standard method (or a custom method in the same *service*) **is** the resource or **contains** the resource, the resource schema for that resource across all methods must be the same.

| Standard method | Request | Response |
| ----------------- | --------- | --------- |
| Create                   | Contains | Resource |
| Get                        | None       | Resource |
| Update                 | Contains | Resource |
| Delete                   | None       | None        |
| List                       | None        | Contains  |

*The table above describes each standard method's relationship to the resource, where "Resource" indicates that the request or response **is** the resource, "Contains" indicates that the request or response has the resource by composition, and "None" indicates that the resource neither **is** nor **is inside** of the request or the response*

Reply:
added! PTAL. I modified things slightly to try to make the table slightly easier to read on first glance.